### PR TITLE
feature/557

### LINKED ISSUE #557: New Airport: KSDF (Louisville Int'l)
Louisville International Airport (IATA: SDF, ICAO: KSDF, FAA LID: SDF) is a public and military use public airport centrally located in the city of Louisville in Jefferson County, Kentucky, United States. The airport is situated on approximately 1,500 acres (6.1 km2) and has three runways. Its IATA airport code, SDF, is based on the airport's former name, Standiford Field. Although it currently does not have regularly-scheduled international passenger flights, it is a port of entry, as it handles numerous international cargo flights.
Over 3.2 million passengers and over 4.7 billion pounds (2,350,000 t) of cargo pass through the airport each year. It is also the third-busiest in the United States in terms of cargo traffic, and seventh-busiest for such in the world. The airport is included in the National Plan of Integrated Airport Systems for 2011–2015, which categorized it as a "primary commercial service" airport since it has over 10,000 passenger boardings (enplanements) per year. As per Federal Aviation Administration records, the airport had 1,650,707 enplanements in 2011, a decrease of 0.02% from 1,651,037 in 2010. The airport is home to Worldport, the worldwide hub of UPS. The Kentucky Air National Guard's 123d Airlift Wing operates C-130 transport aircraft from the co-located Louisville Air National Guard Base.

Taken from Wikipedia

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves openscope/openscope#557 

### Completion Checklist
- [x] terrain file (just one lonely hill at the edge of the map)
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings

---

According to Wikipedia, KSDF has 146,939 yearly operations. Traffic level should fall between 16.8acph and 50.3acph. Currently, there's a total of 20.5acph. 👍

### CODE REVIEW

File: assets/airports/ksdf.json [hunk 1]
```diff
@@ -0,0 +1,1536 @@
...
+        },
+        "DLAMP4": {
+            "icao": "DLAMP4",
+            "name": "Deelamp Four",
+            "entryPoints": {
+                "UKATS": ["UKATS", "SHIRT", "CVG"],
+                "CVG": ["CVG"],
+                "MALDN": ["MALDN", "CVG"]
+            },
+            "body": ["WARSA", "NERVE", "RDSTN", "ZOPOM", "DLAMP"],
```
Comment (erikquinn) [inline assets/airports/ksdf.json]: Missing `HAYGA`

File: assets/airports/ksdf.json [hunk 2]
```diff
@@ -0,0 +1,1536 @@
...
+            "icao": "CHERI3",
+            "name": "Cherry Three",
+            "entryPoints": {
+                "ENL": ["ENL", "ZARDA", "PENTO"],
+                "PXV": ["PXV"],
+                "TTH": ["TTH"]
+            },
+            "body": ["SACKO", "CHERI", "IIU"],
+            "rwy": {
+                "17L": [],
```
Comment (Fechulo) [inline assets/airports/ksdf.json]: Done 👍

File: assets/airports/ksdf.json [hunk 3]
```diff
@@ -0,0 +1,1536 @@
...
+            "name": "Darby Six",
+            "entryPoints": {
+                "HCH": ["HCH", "LVT"],
+                "LOZ": ["LOZ", "UNCKL"],
+                "UNCKL": ["UNCKL"],
+                "VXV": ["VXV"]
+            },
+            "body": ["DARBY", "IIU"],
+            "rwy": {
+                "17L": [],
```
Comment (Fechulo) [inline assets/airports/ksdf.json]: Done 👍

File: assets/airports/ksdf.json (hunk 1 above)
Comment (Fechulo) [inline assets/airports/ksdf.json]: Done 👍

File: assets/airports/ksdf.json [hunk 4]
```diff
@@ -0,0 +1,1536 @@
...
+            "icao": "RDSTN5",
+            "name": "Redstone Five",
+            "entryPoints": {
+                "APE": ["APE", "SHIRT", "CVG"],
+                "CVG": ["CVG"],
+                "ROD": ["ROD", "CVG"]
+            },
+            "body": ["WARSA", "NERVE", "RDSTN", "ZOPOM", "IIU"],
+            "rwy": {
+                "17L": [],
```
Comment (Fechulo) [inline assets/airports/ksdf.json]: Done 👍

**Pull request merged**